action/action.proto

```diff
@@ -37,9 +50,19 @@ message ActionResult {
         COMMAND_DENIED_LANDED_STATE_UNKNOWN = 5;
         COMMAND_DENIED_NOT_LANDED = 6;
         TIMEOUT = 7;
-        UNKNOWN = 8;
+        VTOL_TRANSITION_SUPPORT_UNKNOWN = 8;
+        NO_VTOL_TRANSITION_SUPPORT = 9;
+        UNKNOWN = 1000;
     }
 
     Result result = 1;
     string result_str = 2;
 }
+
+message TakeoffAltitudeResult {
+    float altitude_m = 1;
+}
+
+message MaximumSpeedResult {
+    float speed_mps = 1;
```

Comment:
Its should be `speed_m_s` for [consistency ](https://github.com/dronecore/DroneCore/blob/38e89fc6f5f2038b40a7ae4c8414b48914ad1aca/plugins/action/action.h#L255).

Reply:
Fair point. I'll change it tonight!